MasoDLCPlayer.cs

```diff
@@ -160,17 +167,38 @@ public override void PostUpdateRunSpeeds()
 			}
 			#endregion
 		}
+
 		public override void UpdateDead()
 		{
+            ArcaneShock = false;
 			teslasurge = false;
 			rubberWeapon = false;
 			wristpain = false;
 			creativeblank = false;
 			displayClouds = false;
 			abyssalDrag = false;
 		}
-		public override void UpdateBadLifeRegen()
+
+        public override void PreUpdate()
+        {
+            IsStandingStill = Math.Abs(player.velocity.X) < 0.05 && Math.Abs(player.velocity.Y) < 0.05;

```

Comment:
Any reasons with not just having velocity be completely 0 like most of the other things?

Reply:
Good call